### PR TITLE
Issue #327: fix the toString() of Counter, Gauge, and StatisticsMeter…

### PR DESCRIPTION
**Pull request by gkwan-ibm**
fix the toString() of Counter, Gauge, and StatisticsMeter classes when they are not initialized

### CODE REVIEW

File: dev/com.ibm.ws.monitor/src/com/ibm/websphere/monitor/meters/Counter.java [hunk 1]
```diff
@@ -154,7 +154,10 @@ public CounterReading getReading() {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
-        sb.append("current value = ").append(getCurrentValue());
+        if (terminatedThreadOffset == null)
+            sb.append("not be initialized");
```
Comment (pgunapal) [inline dev/com.ibm.ws.monitor/src/com/ibm/websphere/monitor/meters/Counter.java]: should be "is not initialized"

File: dev/com.ibm.ws.monitor/src/com/ibm/websphere/monitor/meters/Gauge.java [hunk 2]
```diff
@@ -119,14 +125,20 @@ private void updateMinMax(long updated) {
...
         long min = Math.min(current, getMinimumValue());
         long max = Math.max(current, getMaximumValue());
         return new GaugeReading(current, min, max, bounded, lowerBound, upperBound, getUnit());
     }
 
+    @Override
     public String toString() {
+
+        if (currentValue == null)
+            return "not be initialized";
```
Comment (pgunapal) [inline dev/com.ibm.ws.monitor/src/com/ibm/websphere/monitor/meters/Gauge.java]: should be "is not initialized"

File: dev/com.ibm.ws.monitor/src/com/ibm/websphere/monitor/meters/StatisticsMeter.java [hunk 3]
```diff
@@ -353,7 +354,7 @@ public StatisticsReading getReading() {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
-        sb.append(getAggregateStats());
+        sb.append(terminatedThreadStats == null ? "not be initialized" : getAggregateStats());
```
Comment (pgunapal) [inline dev/com.ibm.ws.monitor/src/com/ibm/websphere/monitor/meters/StatisticsMeter.java]: should be "is not initialized"

**Comment by gkwan-ibm** [comment]
- updated monitor fat in the closed liberty so that this problem can be verified
- manually test
    - server.xml with monitor-1.0
    - add com.ibm.websphere.monitor.*=all=enabled to bootstrap.properties 
    - start server
    - use jconsole to view mbean, ConnectionPoolStats MBean should be available under WebSphere

**Comment by gkwan-ibm** [comment]
Personal build result at comment 5 of RTC defect 246663

**Pull request merged**